client/src/types/global-branch.d.ts

```diff
@@ -0,0 +1,3 @@
+declare const XATA_BRANCH: string | undefined;
```

Comment:
Probably move to ``client/src/types/global-cloudflare.d.ts`` as globals are only a cloudflare thing

Reply:
Yep. I got distracted and I thought I didn't made these changes. I'll put it back in the cloudflare globals 👍